src/models/d2Model.ts

```diff
@@ -318,3 +318,21 @@ export function defaultModel(pascalCaseModelName: string): any {
         protected static metadataType = pascalCaseModelName;
     };
 }
+
+export const metadataModels = [
+    DataElementModel,
+    DataElementGroupModel,
+    DataElementGroupSetModel,
+    IndicatorModel,
+    IndicatorGroupModel,
+    IndicatorGroupSetModel,
+    OrganisationUnitModel,
+    OrganisationUnitGroupModel,
+    OrganisationUnitGroupSetModel,
+    ValidationRuleModel,
+    ValidationRuleGroupModel,
+    ProgramIndicatorModel,
+    ProgramIndicatorGroupModel,
+    ProgramRuleModel,
+    ProgramRuleVariableModel,
+];
```

Comment:
We can probably join this metadataModels with the one used in the ``d2ModelFactory`` and only use one for both purposes.

Reply:
fixed!